src/dataset.py

```diff
@@ -100,14 +100,14 @@ def _augment_with_salary(record: RosterRecord) -> Dict[str, str]:
 
 
 def _build_sql_query(queries: List[Tuple]) -> (str, Tuple):
```

Comment:
I think I would write this as 
```suggestion
def _build_sql_query(queries: List[Tuple]) -> Tuple[str, List[str]]:
```